guides/release-process.md

```diff
@@ -136,28 +136,33 @@ In the following instructions, "X.Y.Z" is used to denote the [next version of Cy
     - Merge any release-specific documentation changes into the main release PR.
     - You can view the doc's [branch deploy preview](https://github.com/cypress-io/cypress-documentation/blob/master/CONTRIBUTING.md#pull-requests) by clicking 'Details' on the PR's `netlify-cypress-docs/deploy-preview` GitHub status check.
 
-13. Make the new npm version the "latest" version by updating the dist-tag `latest` to point to the new version:
+13. Create a PR for a new docker image in [`cypress-docker-images`](https://github.com/cypress-io/cypress-docker-images) under `included` for the new cypress version. Note: we use the base image with the Node version matching the bundled Node version. Instructions for updating `cypress-docker-images` can be found [here](https://github.com/cypress-io/cypress-docker-images/blob/master/CONTRIBUTING.md#add-new-included-image). Ensure the docker image is reviewed and has passing tests before preceeding.
```

Comment:
Moved creating the docker PR to be before we make the new tag 'latest'.

Reply:
Just curious, where do we find node version bundled with cypress?

Reply:
It's this: https://github.com/cypress-io/cypress/blob/develop/package.json#L220

It would be the version of node that electron ships.

Reply:
and we update the package.json whenever we update electron